image/user/db/sms_003.sql

```diff
@@ -1,8 +1,8 @@
--- Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+-- Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 -- For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 BEGIN TRANSACTION;
-INSERT OR REPLACE INTO "templates" ("_id","text","lastUsageTimestamp") VALUES (1,'Thanks for reaching out. I can''t talk right now, I''ll call you later',4);
-INSERT OR REPLACE INTO "templates" ("_id","text","lastUsageTimestamp") VALUES (2,'I''ll call you later',3);
-INSERT OR REPLACE INTO "templates" ("_id","text","lastUsageTimestamp") VALUES (3,'I''ll be there in 15 minutes',2);
-INSERT OR REPLACE INTO "templates" ("_id","text","lastUsageTimestamp") VALUES (4,'Give me 5 minutes',5);
```

Comment:
won't this change damage DB in user's phones? afaik we don't have working database migraiton

Reply:
Actually, it looks like it does break DB on phones with the newest OS version :(
This change is on hold until we decide how to deal with DB migration.

Reply:
As we have the UDM already done, the problem is fixed. 
I've added a new field according to the migration policy.